src/components/Header.test.js

```diff
@@ -0,0 +1,27 @@
+// Imports React into our test file.
+import React from 'react'
+
+// Imports Enzyme testing and deconstructs Shallow into our test file.
+import Enzyme, { shallow } from 'enzyme'
+
+// Imports Adapter utilizing the latest react version into our test file so we can run a testing render on any component we may need.
+import Adapter from 'enzyme-adapter-react-16'
+
+// Imports in the component we are going to be testing.
+import Header from './Header'
+
+//Allows us to utilize the adapter we import in earlier, allowing us to call and render a component.
+Enzyme.configure({adapter: new Adapter()})
+
+
+describe("When Header renders", () => {
+    it("displays a NavbarBrand", () => {
+      const header = shallow(<Header />)
+      const headerHeading = header.find("h1").text()
```

Comment:
IF we are goona put the .text() on the variable, let's rename the variable to reflect that. "headerHeadingtext"

Reply:
we will do this on next commit